DataFormats/L1TGlobal/interface/GlobalAlgBlk.h

```diff
@@ -109,9 +101,12 @@ class GlobalAlgBlk
     //Prescale Column
     int m_preScColumn;
 
+    //Menu UUID
+    int m_menuUUID;
+    int m_firmwareUUID;
```

Comment:
I'm glad to see this addition. Do you happen to know if the same UUID should be available from the L1 menu in the EventSetup, to check for consistency ?